src/chopper_count.cpp

```diff
@@ -12,56 +12,86 @@ namespace chopper::count
     parser.info.version = "1.0.0";
+    parser.info.author = "Svenja Mehringer";
+    parser.info.email = "svenja.mehringer@fu-berlin.de";
+    parser.info.short_description = "Estimate the (representative) k-mer count";
 
+    parser.info.description.emplace_back("Estimates the (representative) k-mer count of sequence data by computing "
+                                         "HyperLogLog sketches [1] of the input data. "
+                                         "Additionally, the HLL sketches are stored in a directory (if not disabled) "
+                                         "and can be used in computing an HIBF layout with copper layout.");
+
+    parser.add_subsection("Main Options:");
+    // -----------------------------------------------------------------------------------------------------------------
     parser.add_option(config.data_file,
-                      'f', "data_file",
-                      "Give me a filename to a seqinfo file.",
+                      '\0', "input-file",
+                      "The input must be a file containing paths to sequence data you wish to estimate; one filepath "
+                      "per line. If your file contains auxiliary information (e.g. species ids), your file must be tab-"
+                      "separated.",
                       seqan3::option_spec::required);
+    parser.add_list_item("", "Example file:");
+    parser.add_list_item("", "```");
+    parser.add_list_item("", "/absolute/path/to/file1.fasta");
+    parser.add_list_item("", "/absolute/path/to/file2.fa.gz");
+    parser.add_list_item("", "```");
 
     parser.add_option(config.output_prefix,
-                      'o', "output-prefix",
-                      "chopper count creates a `[PREFIX].counts` file that contains an estimated kmer count for each "
-                      "user bin which is used in chopper layout. Additionally a directory [PREFIX]_sketches is created "
-                      "and one `.hll` file per user bin is computed. The sketch files improve the layout further. "
-                      "Attention: Expects a prefix, thus file extensions are dropped and if given a path (e.g. /tmp/) "
+                      '\0', "output-prefix",
+                      "The program creates a `[PREFIX].counts` file that contains estimated k-mer counts for each "
+                      "sequence file. These counts are used in chopper layout to create an index layout file that can "
+                      "be used by the app raptor to build an HIBF. Additionally, a directory [PREFIX]_sketches is "
+                      "created that will contain one `.hll` file per sequence file. The sketch files improve the "
+                      "layout computation."
+                      "Attention: Expects a prefix, thus file extensions are dropped and if given a path (e.g. /tmp/), "
                       "a default prefix is appended.",
```

Comment:
Can you write `the default prefix [...] is appended.`
We know what the default prefix is, so we can tell the user